src/types/ProposalVersion.ts

```diff
@@ -6,10 +6,13 @@ import type {
 	WritableProposalFieldValueWithProposalVersionContext,
 } from './ProposalFieldValue';
 import type { Writable } from './Writable';
+import type { Source } from './Source';
 
 interface ProposalVersion {
 	readonly id: number;
 	proposalId: number;
+	readonly sourceId: number;
```

Comment:
Yeah you're right that when this is `readonly` it's completely redundant, that said in all other cases we do have both the deep object as well as the ID (this may be a relic of a past when deep objects were conditionally populated). I think it might be worth the redundancy here just for the consistency / not having this be the one case where the foreign key is not included in the readable object, but I'm also OK with taking it out if you disagree.

Also in general I think we might be nearing a point where it's time to start thinking about how to signal which deep objects to provide in a response... Not in this PR though.